frontend/app-development/features/overview/components/News/News.test.tsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
-import { screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
-import { renderWithProviders } from 'app-development/test/testUtils';
-import { APP_DEVELOPMENT_BASENAME, NEWS_EXPIRATION_TIME_IN_DAYS } from 'app-shared/constants';
-import { News } from './News';
+import { render, screen, waitFor } from '@testing-library/react';
+import { NEWS_EXPIRATION_TIME_IN_DAYS } from 'app-shared/constants';
```

Comment:
Is there any reason to place the NEWS_EXPIRATION_TIME_IN_DAYS in `app-shared/constants`? What about placing it as close to the news as possible, like in News.tsx? 😊

Reply:
No strong opinions about this, just figured these kind of constants suited in that file, but no trouble moving it!